docs/book/src/plugins/go-v3-plugin.md

```diff
@@ -0,0 +1,57 @@
+# go/v3
+
+Kubebuilder tool will scaffold the go/v3 plugin by default. This plugin is a composition of the plugins ` kustomize.common.kubebuilder.io/v1` and `base.go.kubebuilder.io/v3`. By using you can scaffold the default project which is a helper to construct sets of [controllers][controller-runtime]. 
```

Comment:
"Kubebuilder tool will scaffold the go/v3 plugin by default." ->  add a new paragraph after this line, and perhaps make it bold.
"By using ..." -> "Using the plugin, you can..."